classes/stock/StockAvailable.php

```diff
@@ -577,7 +577,7 @@ public static function setQuantity($id_product, $id_product_attribute, $quantity
             if ($id_stock_available) {
                 $stock_available = new StockAvailable($id_stock_available);
 
-                $deltaQuantity = -1 * ((int) $stock_available->quantity - (int) $quantity);
+                $deltaQuantity = (int) $quantity - (int) $stock_available->quantity;
```

Comment:
Doesn't this modification change the result ?  (the result used to be negative all the times, it's not the case anymore).

Is this a bug fix ?

If it's just for readability purpose, then there's something wrong with this modification  🤔